pom.xml

```diff
@@ -24,6 +25,12 @@
       <version>3.8.1</version>
```

Comment:
It must be replaced with ${junit.version} from the <properties></properties> section above. Otherwise specifying junit.version there makes no sense if you don't use it.